.coderabbit.yaml

```
@@ -241,9 +404,10 @@
     team_keys: []
   pull_requests:
     scope: auto
+
 code_generation:
   docstrings:
     language: en-US
     path_instructions: []
   unit_tests:
-    path_instructions: []
+    path_instructions: []
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)